docs/examples/example_gs2.py

```diff
@@ -2,9 +2,32 @@
 
 gs2_template = pyrokinetics.template_dir / "input.gs2"
 
-pyro = pyrokinetics.Pyro(gk_file=gs2_template, gk_code="GS2")
+pyro = pyrokinetics.Pyro(gk_file=pyrokinetics.template_dir / "input.gs2")
+print(pyro.local_geometry.B0)
 
+
+pyro = pyrokinetics.Pyro(gk_file=pyrokinetics.template_dir / "input.stella")
```

Comment:
I'm guessing and the below this was not intended to be committed / pushed as clearly not example_gs2 material! (and there's an exit below which is not what you want). Basically should undo all the changes to this file?